lib/src/appengine_internal.dart

```diff
@@ -292,10 +297,15 @@ Future<grpc.Client> _getGrpcClient(
       accessTokenProvider = auth_utils.MetadataAccessTokenProvider();
     }
   }
-  final client = await grpc.connectToEndpoint(Uri.parse(url),
-      accessTokenProvider: accessTokenProvider, timeout: 20);
-  ss.registerScopeExitCallback(client.close);
-  return client;
+  final clientChannel = grpc.ClientChannel(
+    url,
```

Comment:
The constructor here takes a `host` not a `url`, see:
https://pub.dev/documentation/grpc/latest/service_api/ClientChannel/ClientChannel.html

Probably we can just do: `Uri.parse(url).host`

Or we can change the parameter from a URL to a hostname...

Reply:
done